ui/index.html

```diff
@@ -0,0 +1,51 @@
+<html lang="en">
+
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <meta http-equiv="X-UA-Compatible" content="ie=edge">
+        <title>iReporter</title>
+        <!-- include CSS -->
+        <link rel="stylesheet" href="styles.css">
+        <link href="https://fonts.googleapis.com/css?family=Dancing+Script|Lato" rel="stylesheet">
+    </head>
+
+    <body>
+        <nav class="navbar">
+            <div class="container">
+                <ul class="nav">
+                    <li class="nav-item nav-brand">
+                        <a href="index.html">iReporter</a>
+                    </li>
+                </ul>
+                <ul class="nav nav-right">
+                    <li class="nav-item">
+                        <a href="/login.html">Login</a>
+                    </li>
+                    <li class="nav-item">
+                        <a href="/signup.html">Sign Up</a>
+                    </li>
+                </ul>
+            </div>
+        </nav>
+        <header class="banner bg">
+            <div class="container">
+                <div class='banner-title'> Welcome to <span>iReporter</span>!</div>
+                <div class="banner-subtitle">Building a better nation starts with you</div>
+                <a href='login.html'>
+                    <button class="banner-button">WHISTLE BLOW</button>
+                </a>
+            </div>
+        </header>
+        <!-- <section class="landing-content flex-container ">
```

Comment:
remove the commented code if no longer required

Reply:
I am on it.